style.css

```diff
@@ -0,0 +1,223 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+}
+
+main {
+  background-image: url("./images/background.jpeg");
+  background-repeat: no-repeat;
+  background-position: center bottom;
+  z-index: -1;
+  height: 540px;
+}
+
+nav {
+  position: relative;
+  top: 80px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: linear-gradient(#535353, #a6a7ac);
+  width: 70%;
+  height: 36px;
+  margin: auto;
+  padding: 0;
+  border-radius: 10px;
+  font-size: 12px;
+}
+
+#logo {
+  width: 11%;
+  display: flex;
+  justify-content: center;
+}
+
+nav img {
+  width: 22px;
+  height: 22px;
+}
+
+.right-search {
+  position: relative;
+  height: 50%;
+  text-align: center;
+}
+
+.right-search input {
+  height: 100%;
+  width: 80%;
+  background: rgba(0, 0, 0, 0.324);
+  border: none;
+  box-shadow: inset 0 1px 0 1px black;
+  border-radius: 10px;
+}
+
+@import url("//maxcdn.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css");

```

Comment:
Fun fact @import is general slower than doing the regular <link href> in the html file, should you want to change that, that's a good motivation :smiley: but it is not required

Reply:
Erased it , okay :)